Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -496,5 +496,67 @@ protected override void ProcessRecord()
         }
     }
 
-    
+    /// <summary>
+    /// <para type="synopsis">
+    /// Requests that a running BQ Job be canceled.
+    /// </para>
+    /// <para type="description">
+    /// Requests that a job be canceled. This call will return immediately, and the client 
```

Comment:
I would split this up a bit it's a little long winded.

Maybe:
Requests that a job be canceled. This call will return immediately and the client 
is responsible for polling for job status. Canceled jobs may still incur costs. This cmdlet returns a Job object if successful.